src/coreclr/tools/aot/ILCompiler.LLVM/CodeGen/LLVMObjectWriter.cs

```diff
@@ -313,6 +315,16 @@ private void EmitObjectNode(ObjectNode node, ObjectData nodeContents)
             }
         }
 
+        private static bool IsSupportedRelocType(ObjectNode node, RelocType type)
+        {
+            if (node is StackTraceMethodMappingNode)
+            {
+                // Stack trace metadata uses relative pointers, but is currently unused.
```

Comment:
Do we have an issue about deleting it or making it work?

Reply:
Yep - this is #2404.